client/src/pages/EquipmentBrowser/EquipmentBrowser.tsx

```diff
@@ -1,22 +1,85 @@
 import React from 'react';
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
+import 'react-tabs/style/react-tabs.css';
 import styles from './EquipmentBrowser.module.css';
 
+export interface Item {
```

Comment:
We will probably want to move this Item interface somewhere more central in the app, but it's fine here for now

Reply:
I had to refactor this out for my issue #35 so this point is resolved in #47